src/translations/data.json

```diff
@@ -723,1200 +723,1246 @@
+    "hr": "sadašnje",
+    "sr": "садашње",
+    "mk": "сегашно",
     "bg": "настояще"
   },
   "imperfect": {
     "en": "imperfect",
     "isv": "imperfekt",
     "ru": "несов. прошед.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "недокон. минулий",
+    "be": "недаск. мінулы",
+    "pl": "przeszły nedok.",
+    "cs": "minulý nedok.",
+    "sk": "minulý nedok.",
+    "sl": "nepopolni preteklik",
+    "hr": "imperfekt",
+    "sr": "имперфект",
+    "mk": "минато несвршено",
     "bg": "мин. несв."
   },
   "future": {
     "en": "future",
     "isv": "buduče",
     "ru": "будущее",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "майбутній",
+    "be": "будучы",
+    "pl": "przyszły",
+    "cs": "budoucí",
+    "sk": "budúci",
+    "sl": "prihodnjik",
+    "hr": "buduće",
+    "sr": "будуће",
+    "mk": "идно",
     "bg": "бъдеще"
   },
   "perfect": {
     "en": "perfect",
     "isv": "perfekt",
-    "ru": "соверш. прошед.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "ru": "прошлое сов.",
+    "uk": "докон. минулий",
+    "be": "даскан. мінулы",
+    "pl": "przeszły dok.",
+    "cs": "minulý dok.",
+    "sk": "minulý dok.",
+    "sl": "popolni preteklik",
+    "hr": "perfekt",
+    "sr": "перфект",
+    "mk": "минато свршено",
     "bg": "мин. неопр."
   },
   "pluperfect": {
     "en": "pluperfect",
     "isv": "pluskvamperfekt",
-    "ru": "предпрошед.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "ru": "предпрошедщее",
+    "uk": "плюперфектний",
+    "be": "плюсквамперфект",
+    "pl": "zaprzeszły",
+    "cs": "předminulý",
+    "sk": "predminulý",
+    "sl": "predpreteklik",
+    "hr": "pluperfekt",
+    "sr": "плуперфект",
+    "mk": "предминато",
     "bg": "мин. предвар."
   },
   "conditional": {
     "en": "conditional",
     "isv": "kondicional",
     "ru": "сослаг. накл.",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "умовний спосіб",
+    "be": "умоўны лад",
+    "pl": "tryb przypuszcz.",
+    "cs": "podmiňovací",
+    "sk": "podmienečný",
+    "sl": "pogojnik",
+    "hr": "kondicional",
+    "sr": "потенцијал",
+    "mk": "условен начин",
     "bg": "условно накл."
   },
   "infinitive": {
     "en": "infinitive",
     "isv": "infinitiv",
     "ru": "неопределённая форма",
-    "uk": "",
-    "be": "",
-    "pl": "",
-    "cs": "",
-    "sk": "",
-    "sl": "",
-    "hr": "",
-    "sr": "",
-    "mk": "",
+    "uk": "неозначена форма",
+    "be": "інфінітыў",
+    "pl": "bezokolicznik",
+    "cs": "neurčitý",
+    "sk": "neurčitý",
```

Comment:
"neurčitok" ? And in that case maybe also in czech "neurčitek"?

Reply:
sdělano